CHANGELOG.md

```diff
@@ -41,9 +41,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) for all versions `v1.0.0` and beyond (still considered experimental prior to v1.0.0).
 
-## [Unreleased]
-<!-- ## v0.3.0 -->
-* Refactors to adapt to single transaction insertions from workers and reputers.
+## [Unreleased] -- will be v0.4.0
+
+Implements merit based sortition for top N inferers, forecasters, and reputers. Implements fixes for our [June 2024](https://github.com/sherlock-audit/2024-06-allora-judging) Sherlock.xyz audit, including important fixes for determining which topics are considered active.
+
+### Added
+
+* To be filled in before release of v0.4.0
+
+### Removed
+
+* To be filled in before release of v0.4.0
+
+
+### Fixed
+
+* [#544](https://github.com/allora-network/allora-chain/pull/544) Added check against zero-rewards after conversion to cosmosInt
+
+### Security
+
+* See our recent [June 2024](https://github.com/sherlock-audit/2024-06-allora-judging) security audit for a full description of bugs found during that audit.
+
+
+## v0.3.0
+
+Refactors to adapt to single transaction insertions from workers and reputers.
```

Comment:
Should this be added to the "Added" section?

Reply:
I added a summary header so you know it's a summary of the entirety of the version difference